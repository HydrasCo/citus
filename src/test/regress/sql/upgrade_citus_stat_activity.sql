SELECT column_name FROM information_schema.columns WHERE table_name = 'citus_stat_activity'
EXCEPT SELECT column_name FROM information_schema.columns WHERE table_name = 'pg_stat_activity'
ORDER BY 1;

SELECT column_name FROM information_schema.columns WHERE table_name = 'pg_stat_activity'
EXCEPT SELECT column_name FROM information_schema.columns WHERE table_name = 'citus_stat_activity'
ORDER BY 1;
