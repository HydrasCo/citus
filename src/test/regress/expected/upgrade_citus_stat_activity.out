SELECT column_name FROM information_schema.columns WHERE table_name = 'citus_stat_activity'
EXCEPT SELECT column_name FROM information_schema.columns WHERE table_name = 'pg_stat_activity'
ORDER BY 1;
 column_name
---------------------------------------------------------------------
 global_pid
 nodeid
 worker_query
(3 rows)

SELECT column_name FROM information_schema.columns WHERE table_name = 'pg_stat_activity'
EXCEPT SELECT column_name FROM information_schema.columns WHERE table_name = 'citus_stat_activity'
ORDER BY 1;
 column_name
---------------------------------------------------------------------
 (0 rows)

